### PR TITLE
Start converting the wescheme language module to typescript

### PR DESCRIPTION
**Pull request by pcardune**

### CODE REVIEW

File: packages/wescheme-blocks/src/languages/wescheme/ast.tsx [hunk 1]
```diff
@@ -41,23 +55,23 @@ export function LetLikeExpr(from, to, form, bindings, expr, options = {}) {
     pretty(node) {
       return P.lambdaLikeSexpr(
         node.fields.form,
-        P.brackets(node.bindings),
-        node.expr
+        node.fields.bindings,
+        node.fields.expr
       );
```
[the remark this answers is not in the document]
Comment (schanzer) [reply packages/wescheme-blocks/src/languages/wescheme/ast.tsx:60]: This _should_ produce all the bindings, wrapped in parens. I think what you have here will just produce the bindings

File: packages/wescheme-blocks/spec/languages/wescheme/WeschemeParser-test.js [hunk 2]
```diff
@@ -298,6 +298,12 @@ describe("The WeScheme Parser,", function () {
       expect(this.ast[0].fields.expr.type).toBe("functionApp");
       expect(this.ast[0].fields.expr.fields.func.fields.value).toBe("*");
     });
+
+    it("should pretty print", function () {
+      expect(this.ast[0].pretty().display(80).join("\n")).toMatchInlineSnapshot(
+        `"(let* (bindings (define x 1) (define y 2) (define z (+ x y))) (* x y z))"`
+      );
+    });
```
[the remark this answers is not in the document]
Comment (schanzer) [reply packages/wescheme-blocks/spec/languages/wescheme/WeschemeParser-test.js:306]: Yeah, that's not right. It should print the original code, modulo any line breaks introduced by the prettyprinter.

**Comment by pcardune** [comment]
@schanzer ok, updated. how does this version look to you?

**Comment by schanzer** [comment]
Looks great. Thanks!￼

**Pull request merged**